source/brototype.js

```diff
@@ -1,24 +1,20 @@
-(function(module) {
+"use strict";
 
-	"use strict";
+var Bro = require("brototype");
 
-	var Bro = require("brototype");
+var __iCanHaz = Bro.prototype.iCanHaz;
 
-	var __iCanHaz = Bro.prototype.iCanHaz;
+Bro.prototype.iCanHaz1 = function() {
+    var keys = Array.prototype.slice.call(arguments),
+    val,
```

Comment:
@bojan88 Many of the indentation updates have broken variable declaration indentation.. which should remain:
```
var item = something(),
    item2,
    item3;
```